themes/bootstrap3/js/vendor/datatables.min.js

```diff
@@ -0,0 +1,27 @@
+/*
+ * This combined file was created by the DataTables downloader builder:
```

Comment:
Is there a way to obtain/install this file via npm? We're working on using npm for loading Javascript dependencies to make their management more straightforward. It would be nice if we could leverage that mechanism for this new file if possible.

Reply:
I'll check that.

Reply:
Thanks. Assuming NPM can be used, you'll want to update [this package.json](https://github.com/vufind-org/vufind/blob/dev/themes/bootstrap3/package.json) and use [this script](https://github.com/vufind-org/vufind/blob/dev/themes/bootstrap3/tools/copyDependencies.mjs) to copy the code into place.

Reply:
The changes are committed to the repository. I hope I included all necessary files and changes to the mentioned files.